skyrim-platform/src/platform_se/skyrim_platform/ConsoleApi.cpp

```diff
@@ -23,6 +23,7 @@ struct ConsoleCommand
 };
 static std::map<std::string, ConsoleCommand> replacedConsoleCmd;
 static bool printConsolePrefixesEnabled = true;
+static WindowsConsolePrinter* windowsConsolePrinter = NULL;
```

Comment:
theoretically, we should use smart pointers here (like sharedptr or unique_ptr)
WindowsConsolePrinter has destructor that calls FreeConsole
destructor for static var should be called on DLL unload (automatically).

so, could you please use unique_ptr/shared_ptr here, then check what happens if you close game (via menu, via qqq).

if all good, unique_ptr/shared_ptr would be better

Please also make vars g_ prefixed. our original code violates our own style guide. (global vars should look like g_globalVar)